tests/Microsoft.Data.DataFrame.Tests/DataFrameTests.cs

```diff
@@ -1624,7 +1572,7 @@ public void TestClone()
         public void TestColumnCreationFromExisitingColumn()
         {
             DataFrame df = MakeDataFrameWithAllColumnTypes(10);
-            PrimitiveColumn<bool> bigInts = new PrimitiveColumn<bool>("BigInts", df["Int"] > 5);
+            PrimitiveColumn<bool> bigInts = new PrimitiveColumn<bool>("BigInts", df["Int"].PairwiseGreaterThan(5));
```

Comment:
Do you have any thought about `Elementwise` vs. `Pairwise`?

For some reason, I think I lean towards `Elementwise`, because when we do operators (like add, multiply) those will be named ElementwiseMultiply.

Reply:
In Pandas docs, they say element-wise:

[https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.lt.html](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.lt.html)

---
In reply to: [335140660](https://github.com/dotnet/corefxlab/pull/2750#discussion_r335140660) [](ancestors = 335140660)

Reply:
I had the same question. Change it to `Elementwise` now